src/components/Button/styles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export const ButtonReset = styled.button`
+  background-color: white;
+  color: #c94545;
+  font-family: "Nunito", sans-serif;
+  font-weight: 600;
+  border-radius: 3px;
+  border: none;
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  cursor: pointer;
+`;
+export const ButtonDefault = styled.button`
+  font-size: 1.5rem;
+  font-weight: 600;
+  font-family: "Nunito", sans-serif;
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+  color: #c94545;
+  background-color: white;
+  border: none;
+  border-radius: 3px;
+  padding: 1.3rem 3.5rem 1.3rem 3.5rem;
+  max-width: 10rem;
```

Comment:
[suggestion] Usually it is not a good idea to define a width for buttons, it's better to set `width: fit-content` and just add padding to the button so you get the space you need between the button text and its borders.